functions/src/services/freee-api.ts

```diff
@@ -109,7 +109,7 @@ class FreeeApi {
       .then(response => response.data)
       .catch(error => {
         console.error(`Error occured for posting ${path}:`, error.response)
-        return error.response.data as any // TODO return proper response
+        return error.response.data // TODO return proper response
```

Comment:
ts全然わからないんですがここの as anyはどのような役割があったんですか？

Reply:
Typescriptでは普通は返却値の型を必ず指定するのが習慣となっていますが、テンプレートアプリではレスポンスの型を定義していないので、そもそもanyとして返しても意味ない、というのがlintが指摘している点です。

> ERROR: /Users/joshua/work/freee/seminar/test/freee-app-template-firebase/functions/src/services/freee-api.ts:112:16 - This assertion is unnecessary since it does not change the type of the expression.
> npm ERR! code ELIFECYCLE
> npm ERR! errno 2
> npm ERR! functions@ lint: `tslint --project tsconfig.json`
> npm ERR! Exit status 2
> npm ERR! 
> npm ERR! Failed at the functions@ lint script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> npm ERR! A complete log of this run can be found in:
> npm ERR!   /Users/joshua/.npm/_logs/2020-08-26T02_58_04_705Z-debug.log
> Error: functions predeploy error: Command terminated with non-zero exit code2
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! freee-app-template-firebase@1.0.0 deploy: `cd hosting; npm run build; cd ../functions; npm run build; firebase deploy`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the freee-app-template-firebase@1.0.0 deploy script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

Reply:
>This assertion is unnecessary since it does not change the type of the expression.

そもそもanyということか